stock_move_pivot_total_price/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Bernat Obrador
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    product_total_price = fields.Float(
+        compute="_compute_product_total_price",
+        string="Total Product Price",
+        store=True,
+    )
+
+    @api.depends("product_id", "quantity")
```

Comment:
You would also need to depend on product_id.list_price here. However list_price is the wrong field, you should be using lst_price. lst_price is a non stored computed field that takes into account list_price and any price_extra, so it is likely you will need to depend on the dependencies of that.